.github/workflows/release.yml

```
@@ -0,0 +1,54 @@
+        
+    - name: Create Release
+      uses: softprops/action-gh-release@v2
+      with:
+        tag_name: ${{ steps.tag.outputs.tag }}
+        name: ${{ steps.tag.outputs.is_tag == 'true' && steps.tag.outputs.tag || 'Latest Build' }}
+        prerelease: ${{ steps.tag.outputs.is_tag == 'false' }}
+        files: |
+          result/atp/*
+        body: |
+          ${{ steps.tag.outputs.is_tag == 'true' && 'Release' || 'Latest build from main branch' }}
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)